src/schema/v2/partner.ts

```diff
@@ -109,27 +120,35 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           )
 
           interface GravityArgs {
+            artwork_id?: string[]
```

Comment:
`artwork_ids`?

Reply:
this one will stay `artwork_id` though since that's the actual arg we want to pass to Gravity

Reply:
Tricky! Maybe an opportunity for a feature friday to pluralize that parameter someday, I would expect it to be dealing with a singular id string as written.